frontend/packages/dev-console/integration-tests/support/commands/app.ts

```diff
@@ -28,6 +31,25 @@ afterEach(() => {
   // checkErrors();
 });
 
+Cypress.Commands.add("pageTitleShouldContain", (title: string) => {
+  cy.get('[data-test-id ="resource-title"]')
+    .should("be.visible")
+    .and("contain.text", title);
+});
+
+Cypress.Commands.add("alertTitleShouldContain", (alertTitle: string) => {
+  cy.byLegacyTestID("modal-title").should("contain.text", alertTitle);
+});
+
+Cypress.Commands.add('clickNavLink', (path: [string, string]) => {
+  cy.get(`[data-component="pf-nav-expandable"]`) // this assumes all top level menu items are expandable
+    .contains(path[0])
+    .click(); // open top, expandable menu
+  cy.get('#page-sidebar')
+    .contains(path[1])
+    .click();
+});
+
```

Comment:
same as above

Reply:
while pageTitleShouldContain and alertTitleShouldContain are not present, hence retained. However, I removed implementation for clickNavLink which was present in integration-tests-cypress folder